### DISCUSSION TITLE
"Operator" definition

### DISCUSSION THREAD

**Discussion by ghcproject** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#20] [Q&A]
Is it intended that "an entity or group that provides the services offered by the platform to the users of the platform" under the proposed new definition of "operator" are then considered a "virtual asset trading platform" and subject to the VASP Act even if they don't exert managerial control? What's the intent here?

**Comment by FinalTwist** [comment]
from my reading of the change, it seems to me like it's a step process.  For example, the operator is X, if there is no X, then it is Y and if there is no Y, then it is Z.  Seems to me like this is meant to accommodate defi/web3 projects?

If there is a clear operator (X), then Y and Z don't come into play.  Is that how you read it too?

**Reply by Aaron-XReg** [reply]
This seems to be consistent with the IOSCO defi report approach of attempting to find an accountable person in all instances where a defi platform provides financial services. I am still trying to figure out whether the intent is that a foundation company could be considered an operator (see def'n of operator (b)(ii)).

**Comment by JupiterBlock** [comment]
It is proposed that the definition of operator changes as follows:

“**owner**” or “**operator**”, in relation to a virtual asset trading platform, means —
(a) the single entity or group which exerts management control over the platform; 
(b) where a single entity or group which exerts management control is not identifiable, either —
(i) the entity under which the platform operates; or
(ii) the entity under which the platform may contract with third parties; or
(c) where an entity or group under paragraph (a) or (b) is not identifiable, the entity or group that provides the services offered by the platform to the users of the platform."

This is intended to create more clarity via defining a step-by-step approach so that any virtual asset trading platform (which would still not include pure marketplaces or P2P) is tied to at least one entity, and is in line with the MiCA requirements in Europe.

Foundation companies which serve as DAO legal wrappers would not, from my perspective, fall under (b)(ii) unless their objects include the operation of the platform, otherwise simply supporting the development of the protocol generally or ecosystem by giving grants and entering into agreements from that perspective cannot be considered as the platform contracting with third parties. However, @Aaron-XReg your comment makes it clear that unless more clarity is provided on this point in the VASP Act or Regulations, or a law firm carries out a legal analysis for this specific legal risk on a facts and circumstances basis, projects may be well advised to carry out any platform-related operations (contracts, maintenance, etc.) from non-Cayman entities even if they have a Cayman Foundation as a DAO legal wrapper.

**Reply by Aaron-XReg** [reply]
Thanks for your insights @JupiterBlock. I agree that foundation companies should not fall under (b)(ii). It would be an inappropriate from a regulatory perspective to subject a foundation company which serves as a legal wrapper to a DAO to the VASP trading platform regime. 

It is unclear what scenario is contemplated by (b)(ii) that would not already be captured under the other prongs of the definition. 

Subject to any further clarification as to the legislative intent behind the provision **I would recommend that subsection (b)(ii) be deleted from the Owner/Operator definition**.

I would be interested to hear views in support of this deletion or rationale for keeping the provision.

**Reply by FinalTwist** [reply]
So if there is a platform operating, the definition works thus:

1. is there someone directly managing/operating the platform?  
2. if not, is there an entity by which the platform contracts under (for example with clients, third parties)?
3. if not, who is providing the service directly to the client?

I know there have been issues with Binance and a Cayman Holding company here - the Act may be trying to provide clarity on situations like that?

**Reply by CaraProvenance** [reply]
@Aaron-XReg, in a very non-legal, rather operational, response to the proposition of removing subsection (b)(ii), I would suggest rather a carve out that speaks specifically to the role of Foundations Companies in the Virtual Asset sector, as I do believe "contract with third parties" is otherwise a relevant criterion to consider for qualification of Operator status.  

However, I fully agree with the ambiguity around Foundations in this context and believe it merits addressing specifically.  Again caveat, from an operational perspective only, Foundations are not typically used as vehicles for active trading, or indeed controlling/directing of such entities, and we would normally not see the fall in scope in such scenarios.  That said the use of Foundations is nuanced in relation to DAOs/DeFi and new use cases and organizational/corporate governance have emerged.  

It would be beneficial to have a specific reference or consideration of Foundations role in the Virtual Asset eco system, under the proposed owner/operator amendment of the Act. "could" they be considered an operator?
